src/pages/index.js

```diff
@@ -0,0 +1,272 @@
+import { enableValidation, config, resetValidation, disableButton } from "../scripts/validation.js";
+import "./index.css";
+import Api from "../scripts/utils/Api.js";
+import logoSrc from "../images/logo.svg";
+import avatarSrc from "../images/avatar.jpg";
+
+const api = new Api({
+baseUrl: "https://around-api.en.tripleten-services.com/v1",
+headers: {
+authorization: "7ea97be1-ac79-4fc2-bcf3-d29a380283eb",
+"Content-Type": "application/json"
+}
+});
+
+api.getUserInfo()
+  .then((userInfo) => {
+    profileName.textContent = userInfo.name;
+    profileDescription.textContent = userInfo.about;
+    profileAvatar.src = userInfo.avatar;
+  })
+.catch(console.error);
+
+api.getInitialCards().then((cards) => {
+cards.forEach((item) => {
+const cardElement = getCardElement(item);
+cardsList.append(cardElement);
+});
+})
+.catch(console.error);
```

Comment:
You need to format the markup (and code) to show **indentation** and to make the code more readable. You can do that with `prettier` or a combination `SHIFT + ALT + F`

```js
api
  .getUserInfo()
  .then((userInfo) => {
    profileName.textContent = userInfo.name;
    profileDescription.textContent = userInfo.about;
    profileAvatar.src = userInfo.avatar;
  })
  .catch(console.error);

api
  .getInitialCards()
  .then((cards) => {
    cards.forEach((item) => {
      const cardElement = getCardElement(item);
      cardsList.append(cardElement);
    });
  })
  .catch(console.error);

```